pkg/neorpc/result/raw_notary_pool.go

```diff
@@ -0,0 +1,46 @@
+package result
+
+import (
+	"encoding/json"
+
+	"github.com/nspcc-dev/neo-go/pkg/util"
+)
+
+// RawNotaryPool represents a result of `getrawnotarypool` RPC call.
+// The structure consist of `Hashes`. `Hashes` field is a map, where key is
+// the hash of the main transaction and value is a slice of related fallback
+// transaction hashes.
+type RawNotaryPool struct {
+	Hashes map[util.Uint256][]util.Uint256
+}
+
+type rawNotaryPoolAux struct {
```

Comment:
It's not a strict rule, but usually we document these auxiliary structures with `// rawNotaryPoolAux is an auxiliary struct for RawNotaryPool JSON marshalling.`, because the package itself contains a lot of similar structures.

Reply:
Fixed.